mesh/storage.c

```diff
@@ -446,6 +466,7 @@ bool storage_load_nodes(const char *dir_name)
 
 	while ((entry = readdir(dir)) != NULL) {
 		char name_buf[PATH_MAX];
+		char name_with_uuid_buf[PATH_MAX + UUID_LEN];
```

Comment:
This doesn't seem used anywhere. Besides, paths longer than `PATH_MAX` won't work anyway. See `<linux/limits.h>`

Reply:
It is used to delete ..some_path/node_uuid_nr directory when cannot parse neither json nor json.bak files.

Size fixed

Reply:
https://linux.die.net/man/3/dirname

Reply:
Done